### DISCUSSION TITLE
Topbar crashes after installing a layout (OBK 3.00)

### DISCUSSION THREAD

**Discussion by rashidi77** [OpenBangla/OpenBangla-Keyboard#264] [General]
When I install a new layout, the topbar immediately crashes and I have to relaunch it. It's not really a big problem. But it seems weird to me. I launched and tried it from terminal to see if there is any error message but it didn't show any error message during crash.

```
OBK version :  3.00 (compiled from development branch)
Ibus version : 1.5.25
DE : Gnome 41.1
Distro : Fedora 36 Rawhide
```

**Comment by mominul** [comment]
Can you provide the layout?

**Reply by rashidi77** [reply]
It crashes after installing any layout. It's not probably layout specific I think. Anyway, the layout has a copyright issue, so I am not willing to share the layout here.
